tls/s2n_psk.h

```diff
@@ -33,6 +33,24 @@ typedef enum {
     S2N_PSK_DHE_KE,
 } s2n_psk_key_exchange_mode;
 
+typedef enum {
+    S2N_PSK_HMAC_SHA224 = 0,
+    S2N_PSK_HMAC_SHA256,
+    S2N_PSK_HMAC_SHA384,
+} s2n_psk_hmac;
```

Comment:
Why are we including only a small subset of the HMAC Algorithms [supported by s2n](https://github.com/awslabs/s2n/blob/main/crypto/s2n_hmac.h#L22-L32) here ? What about `S2N_HMAC_SHA512`/ `S2N_HMAC_SHA1`?  Is the subset based on the hmac algorithms supported by TLS1.3 ciphers?

Reply:
Yep, we want to give the customers only hmac algorithms for tls1.3.